pkgs/development/compilers/gcc/11/default.nix

```diff
@@ -249,9 +249,9 @@ stdenv.mkDerivation ({
 
   targetConfig = if targetPlatform != hostPlatform then targetPlatform.config else null;
 
-  buildFlags = optional
-    (targetPlatform == hostPlatform && hostPlatform == buildPlatform)
-    (if profiledCompiler then "profiledbootstrap" else "bootstrap");
+  buildFlags =
```

Comment:
Should we wrap a `lib.optional` outside? `makeFlags` is documented as a list of strings.